### PR TITLE
add edit possibility

### PR DESCRIPTION
**Pull request by depfryer**
like that 
https://guilima.github.io/financials-index/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,8 +3,8 @@ node_modules
 npm-debug.log
 .vscode
 .coveralls.yml
-dist/build.js
-dist/build.js.map
+#dist/build.js
```
Comment (charliekassel) [inline .gitignore]: This is here for a reason, I don't want built files on the repository

File: package.json [hunk 2]
```diff
@@ -90,6 +90,7 @@
     "vue-loader": "^12.1.0",
     "vue-style-loader": "^3.0.1",
     "vue-template-compiler": "^2.3.3",
+    "vue-the-mask": "^0.11.0-alpha.2",
```
Comment (charliekassel) [inline package.json]: What is this?

File: src/components/Datepicker.vue [hunk 3]
```diff
@@ -328,13 +334,28 @@ export default {
...
       if (this.selectedDate === null) {
         this.setPageDate()
         return
       }
       this.setPageDate(this.selectedDate)
     },
+    updateDate (event) {
+      let valueDate = this.$refs.inputdatepicker.value
+      let currentYear = this.disabled.from.getFullYear().toString().split('')
+      var regex = new RegExp('^(0[1-9]|[1-2]\\d|3[0-1])\\/(0[1-9]|1[0-2])\\/(19[7-9]\\d|200\\d|20[0-' + currentYear[2] + '][0-' + currentYear[3] + '])$')
```
Comment (charliekassel) [inline src/components/Datepicker.vue]: What is this regex supposed to do?
You should comment anything that is not obvious.

File: package.json (hunk 2 above)
Comment (depfryer) [reply package.json]: it's for add a mask '__/__/_____' when you write a date with keyboard

**Comment by depfryer** [comment]
sorry it's only for french date ^^'

**Pull request closed without merging**